### PR TITLE
Bulk indexing for members/invitations `rebuild_index`

### PR DESCRIPTION
**Pull request by max-moser**
Make the `service.rebuild_index` method use bulk indexing rather than indexing each item individually.
Related to https://github.com/inveniosoftware/invenio-records-resources/pull/419.

### CODE REVIEW

File: invenio_communities/views/api.py [hunk 1]
```diff
@@ -25,9 +25,15 @@ def init(state):
 
     # services
     rr_ext.registry.register(ext.service)
+    rr_ext.registry.register(ext.service.members, service_id="community-members")
```
Comment (ppanero) [inline invenio_communities/views/api.py:28]:
```suggestion
    rr_ext.registry.register(ext.service.members, service_id="members")
```

to be consistent with indexer naming?

File: invenio_communities/members/services/service.py [hunk 2]
```diff
@@ -650,16 +665,12 @@ def rebuild_index(self, identity, uow=None):
...
         """
-        for rec_meta in self.record_cls.model_cls.query.all():
-            rec = self.record_cls(rec_meta.data, model=rec_meta)
-
-            if not rec.is_deleted:
-                self.indexer.index(rec)
-
-        for rec_meta in ArchivedInvitation.model_cls.query.all():
-            rec = ArchivedInvitation(rec_meta.data, model=rec_meta)
+        members = self.record_cls.model_cls.query.filter_by(is_deleted=False).all()
```
Comment (ppanero) [inline invenio_communities/members/services/service.py:668]: same old question about removing the `if`. Feel free to answer to it in only one of the PRs 😅

File: invenio_communities/views/api.py (hunk 1 above)
Comment (max-moser) [reply invenio_communities/views/api.py:28]: will amend the commit with the change on monday

Comment (max-moser) [reply invenio_communities/views/api.py]: updated, and explicitly set the `service_id` for the communities service too, for consistency within this function

**Review by ppanero — APPROVED** [review]
LGTM

# ⚠️ haven't tested yet.

File: invenio_communities/views/api.py [hunk 3]
```diff
@@ -24,10 +24,16 @@ def init(state):
     ext = app.extensions["invenio-communities"]
 
     # services
-    rr_ext.registry.register(ext.service)
+    rr_ext.registry.register(ext.service, service_id="communities")
```
Comment (ppanero) [inline invenio_communities/views/api.py:27]: very minor: would it make sense to do `ext.service.id` (see [here](https://github.com/inveniosoftware/invenio-records-resources/blob/master/invenio_records_resources/services/base/service.py#L76)). Although this is something we have not done in any other registration (maybe this is the point when we start fixing it hehe)

**Pull request merged**